modules/system/console/MixCompile.php

```diff
@@ -221,7 +227,7 @@ protected function createWebpackConfig(string $mixJsPath): void
 
         $config = str_replace(
             ['%base%', '%notificationInject%', '%mixConfigPath%', '%pluginsPath%', '%appPath%', '%silent%'],
-            [$basePath, '', $mixJsPath, plugins_path(), base_path(), (int) $this->option('silent')],
+            [addslashes($basePath), 'mix._api.disableNotifications();', addslashes($mixJsPath), addslashes(plugins_path()), addslashes(base_path()), (int) $this->option('silent')],
```

Comment:
@bennothommo @iamyigitkoc why was it required to add the `mix._api.disableNotifications();` item?